modules/loans/src/lib.rs

```diff
@@ -41,7 +41,7 @@ pub trait Trait: system::Trait {
 
 decl_storage! {
 	trait Store for Module<T: Trait> as Loans {
-		pub Debits get(fn debits): double_map hasher(blake2_256) T::AccountId, hasher(blake2_256) CurrencyIdOf<T> => T::DebitBalance;
+		pub Debits get(fn debits): double_map hasher(blake2_128_concat) CurrencyIdOf<T>, hasher(blake2_128_concat) T::AccountId => (T::DebitBalance, Option<(CurrencyIdOf<T>, T::AccountId)>);
```

Comment:
```suggestion
		pub Debits get(fn debits): double_map hasher(twox_64_concat) CurrencyIdOf<T>, hasher(twox_64_concat) T::AccountId => (T::DebitBalance, Option<(CurrencyIdOf<T>, T::AccountId)>);
```

CurrencyId and AccountId can use `twox_64_concat` as it is not something user can arbitrary crafted to clash the raw key.

Use `blake2_128_concat` for values that user have full control of, like `Balance`, `Vec<u8>`